docs/Guides/Testing.md

```diff
@@ -345,3 +353,140 @@ test('should ...', {only: true}, t => ...)
 
 Now you should be able to step through your test file (and the rest of
 `Fastify`) in your code editor.
+
+
+
+## Plugins
+Let's `cd` into a fresh directory called 'testing-plugin-example' and type `npm init
+-y` in our terminal.
+
+Run `npm i fastify fastify-plugin && npm i tap -D`
+
+To use ESM (`import` statements)
```

Comment:
All other examples in our documentation use CommonJS, please do the same for this example.